wave_docs/api.mdx

```diff
@@ -45,30 +45,31 @@ The endpoint returns the name of the container request made available by Wave.
-| `timestamp`                         | Request submission timestap using ISO-8601.                                                                                                                    |
-| `fingerprint`                       | Request unique fingerprint.                                                                                                                                    |
-| `freeze`                            | The container provisioned will be stored in the specified repository in a permanently.                                                                         |
+| `containerConfig.layers.tarDigest`  | The SHA256checksum of the provided tar file e.g. `sha256:a7c724b02...`.                                                                            |
+| `containerConfig.layers.gzipDigest` | The SHA256 checksum of the provided layer tar gzip file e.g. `sha256:a7c724b02...`.                                                                |
+| `containerConfig.layers.gzipSize`   | The size in bytes of the the provided layer tar gzip file.                                                                                         |
+| `containerFile`                     | Dockerfile used for building a new container encoded in base64 (optional). When provided, the attribute `containerImage` should be omitted.        |
+| `condaFile`                         | Conda environment file encoded as base64 string.                                                                                                   |
+| `spackFile`                         | Spack recipe file encoded as base64 string.                                                                                                        |
+| `containerPlatform`                 | Target container architecture of the built container e.g. `linux/amd64` (optional). Currently only supporting amd64 and arm64.                     |
+| `buildRepository`                   | Container repository where container builds should be pushed e.g. `docker.io/user/my-image` (optional).                                            |
+| `cacheRepository`                   | Container repository used to cache build layers `docker.io/user/my-cache` (optional).                                                              |
+| `timestamp`                         | Request submission timestap using ISO-8601.                                                                                                        |
+| `fingerprint`                       | Request unique fingerprint.                                                                                                                        |
+| `freeze`                            | The container provisioned will be stored in the specified repository in a permanently.                                                             |
```

Comment:
Incomplete sentence.

Reply:
@munishchouhan, can you elaborate on what `freeze` is used for here? Thanks!

Reply:
Freeze requires `buildRepository` to push the build container to a user-defined repository. This will provide the container URL from the user-defined repository, not the generated URL. This URL won't change with simultaneous calls to wave.